src/main/java/com/bugsnag/android/BreadcrumbType.java

```diff
@@ -0,0 +1,48 @@
+package com.bugsnag.android;
+
+/**
+ * Recognized types of breadcrumbs
+ */
+public enum BreadcrumbType {
+    /**
+     * An error was sent to Bugsnag (internal use only)
+     */
+    ERROR ("error"),
+    /**
+     * A log message
+     */
+    LOG ("log"),
+    /**
+     * A manual invocation of `leaveBreadcrumb` (default)
+     */
+    MANUAL ("manual"),
+    /**
+     * A navigation event, such as a window opening or closing
+     */
+    NAVIGATION ("navigation"),
+    /**
+     * A background process such as a database query
+     */
+    PROCESS ("process"),
+    /**
+     * A network request
+     */
+    REQUEST ("request"),
+    /**
+     * A change in application state, such as launch or memory warning
+     */
+    STATE ("state"),
+    /**
+     * A user action, such as tapping a button
+     */
+    USER ("user");
+
+    private final String type;
+
+    BreadcrumbType(String type) {
+        this.type = type;
+    }
+
+    String serialize() { return type; }
```

Comment:
That makes sense and would've made a good error message. Patched in b63033eec62a644aac9e7fd07a8e9577e6af16ac. Non-breaking change since it was package scoped before.